examples/integrations/neo4j/neo4j-mp/src/main/java/io/helidon/examples/integrations/neo4j/mp/domain/MovieRepository.java

```diff
@@ -24,6 +24,9 @@
 import org.neo4j.driver.Value;
 /**
  * The Movies repository.
+ *
+ * @author Michael Simons
+ * @author Dmitry Aleksandov
```

Comment:
Author does not belong to source classes. Copyright should include the original copyright, or a PR should be created with `Co-Authored-By`.